thoth/storages/graph/models.py

```diff
@@ -162,6 +162,7 @@ class PythonPackageVersionEntity(Base, BaseExtension):
         Index(
             "python_package_version_entity_idx",
             "package_name",
+            "id",
```

Comment:
Why `id` was added to this index?

Reply:
It is used for joins

Reply:
I think you want to create a new index on `id`. This way you have added a new key to the existing compound index.

Reply:
Okay, I'll make that change.